src/style.css

```
@@ -0,0 +1,38 @@
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
```

can be simplify, please check in all places
```suggestion
  background: url("./images/star.svg") center no-repeat;
```